client/my-sites/plugins/access-control.js

```diff
@@ -6,32 +6,23 @@ import i18n from 'i18n-calypso';
 /**
  * Internal Dependencies
  */
-import sitesList from 'lib/sites-list';
 import config from 'config';
 import notices from 'notices';
 
-const sites = sitesList();
-
-const hasErrorCondition = ( site, type ) => {
-	const errorConditions = {
-		notMinimumJetpackVersion: site && ! site.hasMinimumJetpackVersion && site.jetpack,
-		notRightsToManagePlugins: sites.initialized && ! sites.canManageSelectedOrAll()
-	};
-	return errorConditions[ type ];
+const getHasRightsToManagePlutins = ( selectedOrAll ) => {
```

Comment:
As I understand, this selector is to be broken down, but it's still useful to note that predicates shouldn't be prefixed with `get`, but rather with interrogative forms such as `is`, `has`, `can`, `should`, etc. As noted elsewhere, the traditional WordPress name for this selector is the more pragmatic `canUser*`.